cloudapi/api.go

```diff
@@ -336,3 +338,95 @@ func (c *Client) ValidateToken() (*ValidateTokenResponse, error) {
 
 	return &vtr, nil
 }
+
+// Organization represents a Grafana Cloud k6 organization
+type Organization struct {
+	ID               int    `json:"id"`
+	GrafanaStackName string `json:"grafana_stack_name"`
+	GrafanaStackID   int    `json:"grafana_stack_id"`
```

Comment:
I have tested the endpoint, and looks like it could return a _null_ `grafana_stack_name` and/or `grafana_stack_id`.

Is that only a particular case for my account? Or an edge case that we should handle?
Because, in such case, I think we should make this `nil`-able, and/or somehow explicitly document that it can be the case.

I'm wondering the wide variety of corner cases we can find one we release this to users 🤔